MovieQuizTests/PresenterConvertTest.swift

```diff
@@ -0,0 +1,51 @@
+import XCTest
+@testable import MovieQuiz
+
+final class MovieQuizViewControllerProtocolMock: MovieQuizViewControllerProtocol {
```

Comment:
Обращу внимание на нейминг, суффикс Protocol обычно добавляют протоколам. В классе такое слово выглядит нелепо. А вот слово Mock в классе уместно. Поэтому стоит поправить так
```suggestion
final class MovieQuizViewControllerMock: MovieQuizViewControllerProtocol {
```